src/main/java/com/zenjava/javafx/maven/plugin/GenerateKeyStoreMojo.java

```diff
@@ -156,56 +162,42 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             }
         }
 
-        if (StringUtils.isEmpty(keyStoreAlias)) {
-            throw new MojoExecutionException("A 'keyStoreAlias' is required to generate a new KeyStore");
-        }
-
-        if (StringUtils.isEmpty(keyStorePassword)) {
-            throw new MojoExecutionException("A 'keyStorePassword' is required to generate a new KeyStore");
-        }
+        checkKeystoreRequiredParameter(keyStoreAlias, "keyStoreAlias");
+        checkKeystoreRequiredParameter(keyStorePassword, "keyStorePassword");
 
         if (keyPassword == null) {
             keyPassword = keyStorePassword;
         }
 
-        StringBuilder domainName = new StringBuilder();
-
-        if (certDomain != null) {
-            domainName.append("cn=").append(certDomain);
-        } else if (project.getOrganization() != null && project.getOrganization().getUrl() != null) {
-            String url = project.getOrganization().getUrl();
-            if (url.startsWith("http://")) {
-                url = url.substring("http://".length());
+        StringBuilder distinguishedName = new StringBuilder();

```

Comment:
I think will be better to use `java.util.StringJoiner`. It can handle separators correctly. In this case you can remove `spacer` argument from `checkAndAddRequiredField`.